src/wallet/wallet2.cpp

```diff
@@ -367,7 +367,7 @@ std::unique_ptr<tools::wallet2> make_basic(const boost::program_options::variabl
 
   // user specified CA file or fingeprints implies enabled SSL by default
   epee::net_utils::ssl_options_t ssl_options = epee::net_utils::ssl_support_t::e_ssl_support_enabled;
-  if (command_line::get_arg(vm, opts.daemon_ssl_allow_any_cert))
+  if (daemon_ssl_allow_any_cert)
```

Comment:
Why not just drop unused `daemon_ssl_allow_any_cert` variable declaration instead?

Reply:
Just did it this way for the sake of consistency. All the other surrounding arguments were assigned to variables before use.